readme.md

```diff
@@ -11,6 +11,12 @@ This is for advanced users. You probably want to use XO directly.
 $ npm install --save-dev eslint-config-xo
 ```
 
+For the `esnext` version you'll also need Babel's eslint parser and plugin:
+
+```
+$ npm install --save-dev babel-eslint eslint-plugin-babel
```

Comment:
Maybe mention below here some of the benefits with using the Babel parser. ES7 stuff, async/await, etc. So the user has some incentive to do the extra work.